package.json

```diff
@@ -228,7 +228,7 @@
     "@types/jest-axe": "^3.5.5",
     "@types/lodash": "^4.14.182",
     "@types/mailparser": "^3.4.0",
-    "@types/node": "^14.14.41",
+    "@types/node": "20.6.3",
```

Comment:
pourquoi on a une version différente du .nvmrc (20 🆚 18) ?

Reply:
Done later, in another PR